pkg/block/test/adapter.go

```diff
@@ -1,89 +0,0 @@
-package adapter_test
-
-import (
-	"context"
-	"fmt"
-	"os"
-	"strings"
-	"testing"
-
-	"github.com/stretchr/testify/require"
-	"github.com/treeverse/lakefs/pkg/block"
-	"github.com/treeverse/lakefs/pkg/block/local"
-	"github.com/treeverse/lakefs/pkg/block/mem"
-)
-
-func TestAdapter(t *testing.T, name string, params interface{}) {
-	a := MakeAdapterByName(t, name, params)
-	t.Run("Adapter_Walk", func(t *testing.T) { testAdapterWalk(t, a) })
-}
-
-func testAdapterWalk(t *testing.T, a block.Adapter) {
```

Comment:
Here too, making sure that the coverage is not lost :)

Reply:
This test will be rewritten once we migrate the walker functionality from the ingest package to the adapters